packages/video_player/video_player_avfoundation/ios/Classes/messages.g.h

```diff
@@ -1,7 +1,7 @@
 // Copyright 2013 The Flutter Authors. All rights reserved.
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
-// Autogenerated from Pigeon (v9.2.4), do not edit directly.
+// Autogenerated from Pigeon (v8.0.0), do not edit directly.
```

Comment:
Can you regenerate the pigeon files? (Make sure to format the files with the format tool afterwards)